pkg/sysregistriesv2/system_registries_v2.go

```diff
@@ -248,6 +248,11 @@ type V2RegistriesConf struct {
 	// potentially use all unqualified-search registries
 	ShortNameMode string `toml:"short-name-mode"`
 
+	// AdditionalLayerStoreAuthHelper is a helper binary that receives
+	// registry credentials pass them to Additional Layer Store for
+	// registry authentication. These credentials are only used for pulling (not pushing).
```

Comment:
```suggestion
	// registry authentication. These credentials are only collected when pulling (not pushing).
```
I think — c/image doesn’t get to say what they are _used for_, the restriction is about the situations in which the helper is called.

Reply:
Fixed.